scripts/release-train-issue-template.md

```diff
@@ -1,91 +0,0 @@
-Release Akka HTTP $VERSION$
```

Comment:
this might be a better place for the template, but all other repos have it in docs